OnboardIos/OnboardIos/Modules/ViewController.m

```diff
@@ -8,35 +8,40 @@
 
 #import "ViewController.h"
 #import "Users.h"
+#import "DetailsViewController.h"
 
 @interface ViewController ()
 
 @end
 
-@implementation ViewController
+@implementation ViewController {
+    Users *_users;
+}
 
 
 @synthesize tableView = _tableview;
-@synthesize users;
 
 - (void)viewDidLoad {
     [super viewDidLoad];
+    
+    //criando a relacao do datasource e delegate com o componente da nib
     _tableview.dataSource = self;
     _tableview.delegate = self;
-    // Do any additional setup after loading the view from its nib.
+
+    
+    // limitar o scroll com relacao aos dados existentes
+    _tableview.bounces = NO;
+    
+    
+    _users = (Users*)[[Users alloc] init];
```

Comment:
give preference to use property and reference variables as self.users and self.tableview . Also tableview is better written as tableView